CONTRIBUTING.md

```diff
@@ -0,0 +1,200 @@
+# Contributing
+
+## Where to start
+
+All contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas are welcome.
+
+The best place to start is to check the [issues](https://github.com/rust-practice/cargo-leet)
+for something that interests you.
+
+## Bug Reports
+
+* Explain what is currently happening and what you expect instead.
```

Comment:
For this one maybe we should point them at the templates? https://github.com/rust-practice/cargo-leet/issues/new/choose